providers/keycloak/src/airflow/providers/keycloak/auth_manager/keycloak_auth_manager.py

```diff
@@ -263,3 +275,15 @@ def _get_headers(access_token):
             "Authorization": f"Bearer {access_token}",
             "Content-Type": "application/x-www-form-urlencoded",
         }
+
+    def get_url_logout(self, **kwargs) -> str:
+        base_url = conf.get("api", "base_url", fallback="/")
+        return urljoin(base_url, f"{AUTH_MANAGER_FASTAPI_APP_PREFIX}/logout")
+
+    def get_redirected_logout_url(self, url: str) -> str:
```

Comment:
Where is it used?

Reply:
Nowhere :) I moved this part to the service and I forgot to delete it, but maybe it should be handled in the auth manager's logout.